### DISCUSSION TITLE
Progress or Activity icon or similar

### DISCUSSION THREAD

**Discussion by whatsupskip** [hassio-addons/addon-sqlite-web#232] [Ideas]
When a long query is run via SQLite Web I find it is difficult to determine if the query is still running. The only why they it seems to be possible is by looking to see if the browser tab has a "x" symbol as distinct from a refresh symbol. It would be good to have something more than that. 

In addition, in the Chrome browser I find that when I long query is running and I switch tabs and then return to the SQLite Web tab that the SQLite section of the screen is black.

**Comment by frenck** [comment]
This is not a feature request the add-on can do anything about. 

Add-ons wrap existing applications, in this case SQLite Web. Please raise such feature requests with the upstream project. Once they implement/add such a thing, it would become available in the add-on as well.

../Frenck
